src/components/Contacts/ContactDetails/ContactDetailsTab/Other/EditContactOtherModal/EditContactOtherModal.test.tsx

```diff
@@ -36,6 +36,11 @@ jest.mock('notistack', () => ({
   },
 }));
 
+jest.mock('i18next', () => ({
+  // this mock makes sure any components using the translate hook can use it without a warning being shown
```

Comment:
This is `i18next.t` right? It's just a JS function not a hook right? Trivial comment...
```suggestion
  // this mock makes sure any components using the translate function can use it without a warning being shown
```